src/Microsoft.Crank.Controller/README.md

```diff
@@ -114,7 +114,7 @@ Options:
     --application.endpoints http://asp-perf-lin:5001
     --application.sdkversion 5.0.100-alpha1-015721 
     --application.dotnetTrace true 
-    --application.collectCounters true 
```

Comment:
Thi ssitll works right?

Reply:
No, it will only be by defining `--application.options.counterProviders System.Runtime`

I could keep the old way (too) for backward compatibility. Only issue is that it's not a server property like it used to be, so in theory it should go in "options". But it's only a logical constraints.

Reply:
You should keep it for now.

Reply:
What about I restore it, and add also `options.collectCounters` to provide the same behavior, and a warning to switch to the new one when it's used.